src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 200px;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  border: 1px solid $background-color;
+  border-radius: 5px;
+
+  &:hover {
+    transition: 0.3s;
+    transform: scale(1.2);
+
+    & .card__title {
+      color: $text-hover-color;
+    }
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__bottom {
+    padding-inline: 16px;
+  }
+
+  &__title {
+    margin-block: 0 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $second-text-color;
+
+    margin-block: 0 16px;
+  }
+
+  &__button {
+    display: block;
+    margin-top: 16px;
+
+    text-decoration: none;
+    background-color: $color-button;
+
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    box-sizing: border-box;
+
+    font-size: 14px;
+    font-weight: 700;
+    color: $second-color-button;
+
+    text-align: center;
+    line-height: 40px;
+
+    &:hover {
+      transition: 0.3s;
+      color: $color-button;
+      background-color: $second-color-button;
+      border: 1px solid $color-button;
```

Comment:
The `transition` property should be set on the `.card__button` class, not on the `:hover` pseudo-class for the same reason as above.